packages/maskbook/src/plugins/ITO/UI/StyledLinearProgress.tsx

```diff
@@ -1,13 +1,26 @@
-import { withStyles, LinearProgress } from '@material-ui/core'
+import { LinearProgress, LinearProgressProps } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
 
-export const StyledLinearProgress = withStyles({
-    root: {
-        height: 8,
-        borderRadius: 5,
-        background: 'rgba(255, 255, 255, 0.3)',
-    },
-    bar: {
-        borderRadius: 5,
-        backgroundColor: '#fff',
-    },
-})(LinearProgress)
+interface StyledLinearProgress extends LinearProgressProps {
+    barcolor?: string
+    backgroundcolor?: string
```

Comment:
`backgroundColor`

Reply:
Warning: React does not recognize the `backgroundColor` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `backgroundcolor` instead. If you accidentally passed it from a parent component, remove it from the DOM element.